src/templates/blog-post.js

```diff
@@ -0,0 +1,95 @@
+import * as React from "react"
+import { graphql } from "gatsby"
+import { GatsbyImage, getImage } from "gatsby-plugin-image"
+import Layout from "../components/layout"
+// prettier-ignore
```

Comment:
Same comment here on the prettier ignore

Reply:
I can remove these before merging. It's a lot more tedious to read and add/remove named imports when they're all on the same line